Ago-Dic-2020/lopez-flores-jorge-luis/Parcial1/Practica 1.py

```diff
@@ -0,0 +1,20 @@
+class Matrix:
+    def __init__(self, mtx_string):
+
+        self.a = [[int(x) for x in i.split()] for i in mtx_string.splitlines()]
+
+    #Declaramos la fila que se mostrará
+    def row(self, indx):
+        self.fila=self.a[:]
+        return self.fila[indx-1]
+
+    #Declaramos la columna que se mostrará
+    def column(self, indx):
+        return[self.a[i][indx-1] for i in range(len(self.a))]
```

Comment:
No hay necesidad de acceder a la lista principal, una vez que se está iterando sobre el elemento interno, se puede acceder al mismo por medio de la copia de la variable

Reply:
También, vale la pena utilizar el `for` como `foreach`, y no como `for` normal con rango